pom.xml

```diff
@@ -1,139 +1,149 @@
-        </dependency>
-        <dependency>
-            <groupId>com.google.code.findbugs</groupId>
-            <artifactId>annotations</artifactId>
-            <version>3.0.0</version>
-            <scope>provided</scope>
-        </dependency>
+      <dependency>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-2.263.x</artifactId>
+        <version>18</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
     </dependencies>
+  </dependencyManagement>
+  
+  <dependencies>
+    <dependency>
+        <groupId>org.jvnet.hudson.plugins</groupId>
+        <artifactId>instant-messaging</artifactId>
+        <version>${version.instant-messaging.plugin}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.jenkins-ci.plugins.workflow</groupId>
+        <artifactId>workflow-step-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.pircbotx</groupId>
+      <artifactId>pircbotx</artifactId>
+      <version>2.0.1</version>
+      <scope>compile</scope>
+      <exclusions>
+        <!-- provides an ancient version, junit plugin is pulling in a more recent one -->
+        <exclusion>
+          <groupId>org.apache.commons</groupId>
+          <artifactId>commons-lang3</artifactId>
+        </exclusion>
+        <!-- provided by jenkins core -->
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+        </exclusion>
+        <!-- provided by jenkins core -->
+        <exclusion>
+          <groupId>com.google.guava</groupId>
+          <artifactId>guava</artifactId>
+        </exclusion>
+        <!-- provided by jenkins core -->
+        <exclusion>
+          <groupId>commons-codec</groupId>
+          <artifactId>commons-codec</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+  </dependencies>
 
-    <repositories>
-        <repository>
-            <id>repo.jenkins-ci.org</id>
-            <url>https://repo.jenkins-ci.org/public/</url>
-        </repository>
-        <repository>
-            <id>jenkins-snapshots</id>
-            <url>https://repo.jenkins-ci.org/content/repositories/snapshots</url>
-            <snapshots>
-                <enabled>true</enabled>
-            </snapshots>
-        </repository>
-    </repositories>
+  <repositories>
+    <repository>
+        <id>repo.jenkins-ci.org</id>
+        <url>https://repo.jenkins-ci.org/public/</url>
+    </repository>
+    <repository>
+        <id>jenkins-snapshots</id>
+        <url>https://repo.jenkins-ci.org/content/repositories/snapshots</url>
+        <snapshots>
+          <enabled>true</enabled>
+        </snapshots>
+    </repository>
+  </repositories>
 
-    <developers>
-        <developer>
-            <id>jimklimov</id>
-            <name>Jim Klimov</name>
-            <email>jimklimov+jenkinsci@gmail.com</email>
-            <timezone>1</timezone>
-        </developer>
```

Comment:
Whyyyy? I was so young! (Also probably due to hard comparisons with the shifted whitespace)

Reply:
Fixed